nemo_skills/dataset/wmt24pp/prepare.py

```
@@ -0,0 +1,57 @@
+from langcodes import Language
+
+from datasets import load_dataset
+from tqdm import tqdm
+
+LANG2CODE = {"de": "de_DE", "es": "es_MX", "fr": "fr_FR", "it": "it_IT", "ja": "ja_JP"}
+
+
+def write_data_to_file(output_file, datasets, tgt_languages):
+    with open(output_file, "wt", encoding="utf-8") as fout:     
+        for tgt_lang in tgt_languages:
+                for src, tgt in zip(datasets[tgt_lang]["source"], datasets[tgt_lang]["target"]):
```

Ruff (0.13.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)